.github/workflows/Create-NewReleases.yml

```diff
@@ -28,27 +28,33 @@ jobs:
       - name: Determine Next Version Tag
         id: nextver
         run: |
-          TAG=$(git tag --sort=-v:refname | head -n 1)
-          if [[ $TAG == '' ]]; then 
-            TAG='v0.1.0' # Start from v0.1.0 if no tags are found
-          fi
-          MAJOR=$(echo $TAG | cut -d '.' -f 1 | cut -c 2-)
-          MINOR=$(echo $TAG | cut -d '.' -f 2)
-          PATCH=$(echo $TAG | cut -d '.' -f 3)
+          if [[ -f version.txt ]]; then
+            NEXT_TAG=$(cat version.txt)
+          else
+            CURRENT_TAG=$(git tag --sort=-v:refname | head -n 1)
+            if [[ $TAG == '' ]]; then 
```

Comment:
Fixed variable name from "**TAG**"  to  "**CURRENT_TAG**"  (fix included as part of PR #178).

Reply:
Smart man, totally let this over from the old solution. My bad! Wil review the PR now 😄